code/handlers/trackservers.q

```diff
@@ -235,6 +235,8 @@ addprocs:{[connectiontab;procs;connect]
 
 // addprocscustom is to allow bespoke extensions when adding processes
 addprocscustom:{[connectiontab;procs]}
```

Comment:
The error trap should work with this.  (it just gave a projection before)
addprocscustom:{$[x=0b;{[connectiontab;procs]};x]}[@[value;`.servers.addprocscustom;0b]]
is there anything more I need to do on it ?